intel-sgx/pcs/src/tcb_evaluation_data_numbers.rs

```diff
@@ -114,6 +114,10 @@ impl RawTcbEvaluationDataNumbers {
         }
     }
 
+    pub fn filename() -> &'static str {
```

Comment:
Let's return a `String` so we have more flexibility for future alternative implementations of this method.